mmv1/templates/terraform/examples/managedkafka_topic_basic.tf.erb

```diff
@@ -0,0 +1,34 @@
+resource "google_managed_kafka_cluster" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Remove <%= ctx[:primary_resource_id] %> here and replace it with a string like "cluster"